internal/stack/logs.go

```diff
@@ -15,13 +15,19 @@ import (
 )
 
 func dockerComposeLogs(serviceName string, snapshotFile string) ([]byte, error) {
+	appConfig, err := install.Configuration()
+	if err != nil {
+		return nil, errors.Wrap(err, "can't read application configuration")
+	}
+
 	p, err := compose.NewProject(DockerComposeProjectName, snapshotFile)
 	if err != nil {
 		return nil, errors.Wrap(err, "could not create docker compose project")
 	}
 
 	opts := compose.CommandOptions{
 		Env: newEnvBuilder().
+			withEnvs(appConfig.StackImageRefs(install.DefaultStackVersion).AsEnv()).
```

Comment:
There are still these warnings:
```
WARN[0000] The "PROFILE_NAME" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PROFILE_NAME" variable is not set. Defaulting to a blank string.
```

but that missing environment variable does not make fail the command as these ones:
```
WARN[0000] The "ELASTIC_AGENT_IMAGE_REF" variable is not set. Defaulting to a blank string. 
WARN[0000] The "KIBANA_IMAGE_REF" variable is not set. Defaulting to a blank string. 
WARN[0000] The "ELASTIC_AGENT_IMAGE_REF" variable is not set. Defaulting to a blank string.
WARN[0000] The "ELASTICSEARCH_IMAGE_REF" variable is not set. Defaulting to a blank string.
```
As an eaxmple of error:
> service "elasticsearch" has neither an image nor a build context specified: invalid compose project

Reply:
Added that missing env. variable in https://github.com/elastic/elastic-package/pull/1252/commits/a7c35c788f9a7b30a05a2ca65aaecc242095a9eb